app/main.py

```
@@ -1 +1,18 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    parts = command.strip().split()
+
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError("Invalid command. Expected format: "
+                         "mv source destination")
+
+    src, dest = parts[1], parts[2]
+
+    dest_dir = os.path.dirname(dest)
+    if dest_dir and not os.path.exists(dest_dir):
+        os.makedirs(dest_dir)
+
+    shutil.move(src, dest)
```

Line 18: Final call to `shutil.move(src, dest)` both bypasses required manual copy/remove semantics and may move directories (contrary to the requirement to support only moving files). Before moving, check that `src` is a file (`os.path.isfile(src)`) and raise an appropriate error if not; then perform manual copy and `os.remove(src)` after successful write. This ensures the function only moves files and propagates any IO errors instead of silently doing the wrong operation.